app/models/page_sequence.rb

```diff
@@ -19,6 +19,7 @@ class PageSequence
     "full-name",
     "address",
     "date-of-birth",
+    "gender",
```

Comment:
Should this also be `payroll-gender`?

Reply:
We kept the slug for the page the same. I suggest we look to change it separately in the interest of getting this merged and unblocking the dependent work.